composer.json

```diff
@@ -92,6 +92,7 @@
 		],
 		"test": [
 			"composer validate",
+			"./bin/console doctrine:migrations:migrate --env=test --no-interaction",
```

Comment:
It doesn't really matter because this runs pretty quickly, but this feels like it should be paired with a Composer script that runs phpunit (or that should be added here). As it is, this command isn't required to run the rest of `test`, but we'd be assuming people would run `composer test` before `./bin/phpunit` (which they probably would, so this doesn't really matter...).

Reply:
Yeah I added it here thinking "why not". It indeed doesn't really make sense given this script does not run the tests that depend on the migrations, as you say :)

I'll just remove it. The README changes are enough for now, I think.